web/src/components/shared/Chat/Chat.tsx

```diff
@@ -6,19 +6,19 @@ import SendIcon from 'web/src/assets/svg/SendIcon.svg';
 import AttachIcon from 'web/src/assets/svg/AttachIcon.svg';
 import { IconButton } from 'web/src/components/IconButton/IconButton';
 import { Spinner } from 'web/src/components/ui/Spinner';
-import { useSharedT } from 'web/src/components/shared/Adapter';
-import * as styles from './Chat.css';
+import { SharedTranslateFn } from 'web/src/components/shared/sharedI18n';
+import * as styles from 'web/src/components/shared/Chat/Chat.css';
```

Comment:
```suggestion
import * as styles from './Chat.css';
```
так можно, файлы в одной папке